frontend/encore/server-data.rst

```diff
@@ -6,7 +6,7 @@ In Symfony applications, you may find that you need to pass some dynamic data
 dynamic configuration is by storing information in ``data`` attributes and reading
 them later in JavaScript. For example:
 
-.. code-block:: twig
+.. code-block:: html-twig
```

Comment:
`html+twig` (see the failing build)

Reply:
Correct! I'm fixing this while merging. Thanks.